src/control/tools/PdfTextSelection.cpp

```diff
@@ -0,0 +1,140 @@
+#include "PdfTextSelection.h"
+
+#include "control/Control.h"
+
+PdfTextSelection::PdfTextSelection(double x, double y, XojPageView* view) {
+    this->sx = x;
+    this->sy = y;
+    this->ex = x;
+    this->ey = y;
+
+    this->view = view;
+
+    this->isFinalized = false;
+    this->isFinished = false;
```

Comment:
(<i>Really minor:</i> Feel free to ignore!) Personally, I think the following is a bit easier to read. :)

```suggestion
PdfTextSelection::PdfTextSelection(double x, double y, XojPageView* view):
        sx(x),
        sy(y),
        ex(x),
        ey(y),

        view(view),
        isFinalized(false),
        isFinished(false) {
```

Reply:
Yes, I would move these initializations to the declaration line as suggested above. `clang-format` will reformat these to be on one line, making the code clearer.